lib/pay.rb

```diff
@@ -98,7 +98,7 @@ def self.mailer
     if ActionMailer::Base.respond_to?(:email_address_with_name)
       ActionMailer::Base.email_address_with_name(params[:pay_customer].email, params[:pay_customer].customer_name)
```

Comment:
Should we also do this for `ActionMailer`?

Reply:
We probably should for consistency/clarity if nothing else.